### PR TITLE
importinto 8.0 change & DXF enhancement

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16672
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-import-into.md [hunk 1]
```diff
@@ -22,22 +25,34 @@ For TiDB Self-Hosted, `IMPORT INTO` supports importing data from files stored in
...
-- Known issue: the `IMPORT INTO` task might fail if the PD address in the TiDB node configuration file is inconsistent with the current PD topology of the cluster. This inconsistency can arise in situations such as that PD was scaled in previously, but the TiDB configuration file was not updated accordingly or the TiDB node was not restarted after the configuration file update.
+- All `IMPORT INTO` tasks that are created when [TiDB Distributed eXecution Framework (DXF)](/tidb-distributed-execution-framework.md) is not enabled run directly on the nodes where the tasks are submitted, and these tasks will not be scheduled for execution on other TiDB nodes even after DXF is enabled later. After DXF is enabled, only newly created `IMPORT INTO` tasks that import data from S3 or GCS will be automatically scheduled or failover to other TiDB nodes for execution.
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:47]:
```suggestion
- All `IMPORT INTO` tasks that are created when [TiDB Distributed eXecution Framework (DXF)](/tidb-distributed-execution-framework.md) is not enabled run directly on the nodes where the tasks are submitted, and these tasks will not be scheduled for execution on other TiDB nodes even after DXF is enabled later. After DXF is enabled, only newly created `IMPORT INTO` tasks that import data from S3 or GCS are automatically scheduled or failover to other TiDB nodes for execution.
```

File: sql-statements/sql-statement-import-into.md [hunk 2]
```diff
@@ -5,39 +5,42 @@ summary: An overview of the usage of IMPORT INTO in TiDB.
...
 - For TiDB Self-Hosted, the TiDB [temporary directory](https://docs.pingcap.com/tidb/stable/tidb-configuration-file#temp-dir-new-in-v630) is expected to have at least 90 GiB of available space. It is recommended to allocate storage space that is equal to or greater than the volume of data to be imported.
-- One import job supports importing data into one target table only. To import data into multiple target tables, after the import for a target table is completed, you need to create a new job for the next target table.
+- One import job supports importing data into one target table only.
 - `IMPORT INTO` is not supported during TiDB cluster upgrades.
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:22]: Tips for reviewers: L23 was the same as the old L38

File: sql-statements/sql-statement-import-into.md [hunk 3]
```diff
@@ -5,39 +5,42 @@ summary: An overview of the usage of IMPORT INTO in TiDB.
...
+- Ensure that the data to be imported does not contain any records with primary key or non-null unique index conflicts. Otherwise, the conflicts can result in import task failures.
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:23]: Tips for reviewers: L23 was the same as the old L40

File: sql-statements/sql-statement-import-into.md [hunk 4]
```diff
@@ -5,47 +5,49 @@ summary: An overview of the usage of IMPORT INTO in TiDB.
...
 - Known issue: the `IMPORT INTO` task might fail if the PD address in the TiDB node configuration file is inconsistent with the current PD topology of the cluster. This inconsistency can arise in situations such as that PD was scaled in previously, but the TiDB configuration file was not updated accordingly or the TiDB node was not restarted after the configuration file update.
 
+### `IMPORT INTO ... FROM FILE` restrictions
+
+- For TiDB Self-Hosted, `IMPORT INTO` supports importing data within 10 TiB. For [TiDB Dedicated](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-dedicated), `IMPORT INTO` supports importing data within 50 GiB.
+- The execution of `IMPORT INTO ... FROM FILE` blocks the current connection until the import is completed. To execute the statement asynchronously, you can add the `DETACHED` option.
+- Up to 16 `IMPORT INTO` tasks can run simultaneously on each cluster (see [TiDB Distributed eXecution Framework (DXF) usage limitations](/tidb-distributed-execution-framework.md#usage-limitations)). When a cluster lacks sufficient resources or reaches the maximum number of tasks, newly submitted import tasks are queued for execution.
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:30]:
```suggestion
- Up to 16 `IMPORT INTO` tasks can run simultaneously on each cluster (see [TiDB Distributed eXecution Framework (DXF) usage limitations](/tidb-distributed-execution-framework.md#limitation)). When a cluster lacks sufficient resources or reaches the maximum number of tasks, newly submitted import tasks are queued for execution.
```

File: sql-statements/sql-statement-import-into.md [hunk 5]
```diff
@@ -5,47 +5,49 @@ summary: An overview of the usage of IMPORT INTO in TiDB.
...
+
+### `IMPORT INTO ... FROM SELECT` restrictions
+
+- `IMPORT INTO ... FROM SELECT` can only be executed on the TiDB node that the current user is connected to, and it blocks the current connection until the import is complete.
+- `IMPORT INTO ... FROM SELECT` only supports two [import options](#withoptions): `THREAD` and `DISABLE_PRECHECK`.
+- `IMPORT INTO ... FROM SELECT` does not support the task management statements such as `SHOW IMPORT JOB(s)` and `CANCEL IMPORT JOB <job-id>`.
+- The [temp directory](/tidb-configuration-file.md#temp-dir-introduced-from-v630-version) of TiDB requires sufficient space to store the entire query result of the `SELECT` statement (configuring the `DISK_QUOTA` option is not supported currently).
```
Comment (qiancai) [inline sql-statements/sql-statement-import-into.md:41]:
```suggestion
- The [temporary directory](https://docs.pingcap.com/tidb/stable/tidb-configuration-file#temp-dir-new-in-v630) of TiDB requires sufficient space to store the entire query result of the `SELECT` statement (configuring the `DISK_QUOTA` option is not supported currently).
```

File: sql-statements/sql-statement-import-into.md [hunk 6]
```diff
@@ -244,15 +268,21 @@ And assume that the target table schema for the import is `CREATE TABLE t(id int
 IMPORT INTO t(id, name, @1) FROM '/path/to/file.csv' WITH skip_rows=1;
 ```
 
-### Import multiple data files using the wildcard `*`
+#### Import multiple data files using wildcards
 
 Assume that there are three files named `file-01.csv`, `file-02.csv`, and `file-03.csv` in the `/path/to/` directory. To import these three files into a target table `t` using `IMPORT INTO`, you can execute the following SQL statement:
 
 ```sql
 IMPORT INTO t FROM '/path/to/file-*.csv'
```
Comment (lilin90) [inline sql-statements/sql-statement-import-into.md:276]:
```suggestion
IMPORT INTO t FROM '/path/to/file-*.csv';
```

File: sql-statements/sql-statement-import-into.md [hunk 7]
```diff
@@ -244,15 +268,21 @@ And assume that the target table schema for the import is `CREATE TABLE t(id int
...
 ```
 
-### Import data files from Amazon S3 or GCS
+If you only need to import `file-01.csv` and `file-03.csv` into the target table, execute the following SQL statement:
+
+```sql
+IMPORT INTO t FROM '/path/to/file-0[13].csv'
```
Comment (lilin90) [inline sql-statements/sql-statement-import-into.md:282]:
```suggestion
IMPORT INTO t FROM '/path/to/file-0[13].csv';
```

**Review by lilin90 — APPROVED** [review]
Rest LGTM

**Pull request merged**